django/db/models/expressions.py

```diff
@@ -10,6 +10,20 @@
 from django.utils.functional import cached_property
 
 
+class SQLiteNumericMixin:
+    """
+    Some expressions with output_field=DecimalField() must be casted to
+    numeric to be properly filtered.
+    See https://code.djangoproject.com/ticket/18247.
```

Comment:
I decided to remove ticket reference because now I'm not sure if some additional info is needed here.